src/common/legacy_config_opts.h

```diff
@@ -844,26 +843,6 @@ OPTION(kinetic_user_id, OPT_INT) // kinetic user to authenticate as
 OPTION(kinetic_hmac_key, OPT_STR) // kinetic key to authenticate with
 OPTION(kinetic_use_ssl, OPT_BOOL) // whether to secure kinetic traffic with TLS
 
-
-OPTION(rocksdb_separate_wal_dir, OPT_BOOL) // use $path.wal for wal
-SAFE_OPTION(rocksdb_db_paths, OPT_STR)   // path,size( path,size)*
```

Comment:
this is actually a list of tuples, like `(${path},${size}[; \t])*${path},${size}`, for example, 
```
/path/to/rocksdb,1024;/path/to/another-rocksdb,2048
```
or 
```
/path/to/rocksdb,1024
```
could you at least copy the comments like `"path,size( path,size)*"` to `.set_description()` of the new style option, otherwise the descriptions lost when we are cleanup the legacy options.